extractor/filesystem/language/haskell/stacklock/testdata/valid

```diff
@@ -37,4 +37,5 @@ snapshots:
     sha256: 1e32b51d9082fdf6f3bd92accc9dfffd4ddaf406404427fb10bf76d2bc03cbbb
     size: 720263
     url: https://raw.githubusercontent.com/commercialhaskell/stackage-snapshots/master/lts/22/41.yaml
-  original: lts-22.41
\ No newline at end of file
+  original: lts-22.41
+  
```

Comment:
There is some trailing whitespace here - the file should end in a newline character.